doc/release/upcoming_changes/22575.compatibility.rst

```diff
@@ -0,0 +1,8 @@
+``np.pad`` with ``mode=wrap`` now returns expected results
+----------------------------------------------------------
+
+Codes based on earlier version of ``pad`` with ``wrap`` will 
+break when padding size is larger than initial array.
```

Comment:
```suggestion
Code based on earlier version of ``pad`` that uses  ``mode="wrap"`` will return
different results when the padding size is larger than initial array.
```
I think together with the following sentence, this explains more precisely how the code will break.